### PR TITLE
Updated for Bootsrtrap 5

### PR DESCRIPTION
**Pull request by planetoftheweb**

### CODE REVIEW

**Comment by planetoftheweb** [comment]
Hello team. I'm thinking about updating to Bootstrap 5. What do you think?

**Comment by jojohumphreys** [comment]
Sounds great. I'm all for it.

File: docs/media.html [hunk 1]
```diff
@@ -93,10 +88,6 @@ <h4 class="">Media Three</h4>
       integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
```
Comment (jojohumphreys) [inline docs/media.html:88]: Looks like you left the jQuery code, you can safely remove that in all files.

**Review by jojohumphreys — CHANGES_REQUESTED** [review]
This needs a bit of an update to remove jQuery

File: docs/media.html (hunk 1 above)
Comment (planetoftheweb) [reply docs/media.html:88]: That's a great catch. I went ahead and fixed it.

**Pull request merged**